cnxdb/archive-sql/get-featured-links.sql

```diff
@@ -15,13 +15,18 @@ FROM (SELECT
     m.moduleid AS legacy_id,
     m.version AS legacy_version,
     a.html AS abstract,
-    '/resources/' || f.sha1 AS "resourcePath",
+    '/resources/' || files.sha1 AS "resourcePath",
```

Comment:
The database is not an authority on our routes. I'd suggest we think about the separation of concerns here. URL build should happen at the representation layer, which happens to be our web app.